test-lib.sh

```diff
@@ -72,6 +72,7 @@ function ct_check_envs_set {
     var_name=$(echo "$variable" | awk -F= '{ print $1 }')
     stripped=$(echo "$variable" | awk -F= '{ print $2 }')
     filtered_envs=$(echo "$check_envs" | grep "^$var_name=")
+    [ -z "$variable" ] && continue
     [ -z "$filtered_envs" ] && { echo "$var_name not found during \` docker exec\`"; return 1; }
```

Comment:
I would test for emptiness directly after entering the while loop (before actually using `$variable`):

```suggestion
    [ -z "$variable" ] && continue
    var_name=$(echo "$variable" | awk -F= '{ print $1 }')
    stripped=$(echo "$variable" | awk -F= '{ print $2 }')
    filtered_envs=$(echo "$check_envs" | grep "^$var_name=")
    [ -z "$filtered_envs" ] && { echo "$var_name not found during \` docker exec\`"; return 1; }
```